.travis.yml

```diff
@@ -14,6 +14,7 @@ addons:
 
 env:
     global:
+        - SYMFONY_VERSION=4.4
```

Comment:
We have to put the version somewhere, so here we are, in `.travis.yml`.
To be bumped when creating new branches of course.